scripts/globals/gambits.lua

```diff
@@ -75,6 +75,8 @@ ai.select =
     BEST_AGAINST_TARGET = 6,
     BEST_SAMBA          = 7,
     HIGHEST_WALTZ       = 8,
+    ENTRUSTED           = 9,
+    BESTINDI            = 10,
```

Comment:
If this is two words, can it be `BEST_INDI`?

Reply:
yes you'r right, it should be